docker/pypi/wmagent-base/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM registry.cern.ch/cmsweb/oracle:21_5-stable as oracle
+FROM registry.cern.ch/cmsweb/dmwm-base:pypi-20230525
+MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+
+# Install basic OS package dependencies
+RUN apt-get update
+RUN apt-get install -y libmariadb-dev-compat libmariadb-dev apache2-utils cron mariadb-server myproxy voms-clients rlwrap libaio1 && apt-get clean
+
+# Install some debugging tools
+RUN apt-get install -y hostname net-tools iputils-ping procps && apt-get clean
+
+# copy oracle client:
+COPY --from=oracle /usr/lib/oracle /usr/lib/oracle
+ENV LD_LIBRARY_PATH=/usr/lib/oracle
+ENV PATH=$PATH:/usr/lib/oracle
+ENV PKG_CONFIG_PATH=/usr/lib/oracle
+
+# Add all deployment needed directories, install and run scripts
+ADD etc /data/etc
```

Comment:
Please do not hardcode absolute paths, but do use relative paths originating everything from a single $WMA_ROOT_DIR as it was done in
https://github.com/dmwm/CMSKubernetes/pull/1364/files#diff-6d65f482db9361d53e7c64522f0be43dc64fde7f6999426c0657cb064a74283eR23-R37

We have seen already how we easily start juggling with paths and we eventually end up with few of them unlinked to a single root. And this  is a path to unmanageable situation in the future.

This is also needed for the scripts that are to implement an equivalent to the Docker environment.

Reply:
same for all lines bellow containing `/data` as hardwired path